bin/tessel-2.js

```diff
@@ -69,6 +69,44 @@ parser.command('provision')
   })
   .help('Authorize your computer to control the USB-connected Tessel');
 
+parser.command('reboot')
+  .callback(callControllerCallback('reboot'))
+  .option('time', {
+    abbr: 't',
+    metavar: 'SEC',
+    required: false,
+    help: 'Delay interval until reboot (cannot be stopped!)'
+  })
+  .option('delay', {
+    abbr: 'd',
+    metavar: 'SEC',
+    required: false,
+    help: 'Same as --time'
+  })
+  .option('force', {
+    abbr: 'f',
+    required: false,
+    help: 'don\'t go through init'
+  })
+  .option('nosync', {
+    abbr: 'n',
+    required: false,
+    help: 'Do not sync'
+  })
+  .option('name',   {
+    required: true,
+    metavar: 'TESSEL',
+    help: 'Tessels name or IP'
+  })
+  .option('key', {
+    abbr: 'i',
+    default: '~/.tessel/id_rsa', // TODO: replace this by global var
+    required: false,
+    metavar: 'PATH',
+    help: 'SSH-Key to login into your Tessel. Default is:'
+  })
```

Comment:
About the `key` option to become a default option in `makeCommand` is worth to think about, because the discussed things behind `t2 select` make this a fitting solution.